web/templates/invite_user_modal.hbs

```diff
@@ -4,12 +4,7 @@
     <div class="alert" id="dev_env_msg"></div>
     {{/if}}
     {{#if (not user_has_email_set) }}
-    <div class="tip">
-        {{#tr}}
-            You must <z-link>configure your email</z-link> to access this feature.
-            {{#*inline "z-link"}}<a href="/help/demo-organizations#configure-email-for-demo-organization-owner" target="_blank" rel="noopener noreferrer">{{> @partial-block }}</a>{{/inline}}
-        {{/tr}}
-    </div>
+    <div class="invite-banner-container"></div>
```

Comment:
How about `demo-organization-add-email-banner`? That way it's clear that this is a demo organization feature if someone's reading the code.